File: app/src/main/java/projects/csce/evence/SecondActivity.java
package projects.csce.evence;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import javax.inject.Inject;

import okhttp3.OkHttpClient;

public class SecondActivity extends AppCompatActivity
{
    @Inject
    OkHttpClient client;

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ((BaseApplication) getApplication()).getInjector().inject(this);

        client.authenticator();
    }
}

File: app/src/main/java/projects/evenceteam/evence/view/ui/SecondActivity.kt
package projects.evenceteam.evence.view.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import projects.evenceteam.evence.databinding.ActivityLoggedInBinding

class SecondActivity : AppCompatActivity()
{
    private var binding: ActivityLoggedInBinding? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityLoggedInBinding.inflate(layoutInflater)
        setContentView(binding!!.root)
    }
}